spec/models/book_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+RSpec.describe Book, type: :model do
+  describe 'Table columns' do
+    it { is_expected.to have_db_column(:name).of_type(:string) }
+    it { is_expected.to have_db_column(:price).of_type(:decimal) }
+    it { is_expected.to have_db_column(:width).of_type(:decimal) }
+    it { is_expected.to have_db_column(:height).of_type(:decimal) }
+    it { is_expected.to have_db_column(:length).of_type(:decimal) }
+    it { is_expected.to have_db_column(:description).of_type(:text) }
+    it { is_expected.to have_db_column(:quantity).of_type(:integer) }
+    it { is_expected.to have_db_column(:year_of_publication).of_type(:integer) }
```

Comment:
nit: in future add with_options (like null: false etc)

Reply:
ok